### PR TITLE
Fix Jaxis

### LINKED ISSUE #1159: Jaxis, the Troublemaker, the Gained Abilities shouldn't be copyable
`Jaxis, the Troublemaker` and make similar that does Copy the Permanents and say the token **gains** abilities, these abilities shouldn't be copyable when something else does try to make copies of that token.

### PR DESCRIPTION
**Pull request by tool4ever**
Closes #1159

### CODE REVIEW

**Comment by tool4ever** [comment]
Ok, found some others

I think ETB then gain like it's done currently makes sense

File: forge-gui/res/cardsfolder/j/jaxis_the_troublemaker.txt [hunk 1]
```diff
@@ -2,7 +2,9 @@ Name:Jaxis, the Troublemaker
 ManaCost:3 R
 Types:Legendary Creature Human Warrior
 PT:2/3
-A:AB$ CopyPermanent | Cost$ R T Discard<1/Card> | ValidTgts$ Creature.Other+YouCtrl | TgtPrompt$ Select another target creature you control | AddKeywords$ Haste | AddTriggers$ Dies | AddSVars$ TrigDraw | AtEOT$ Sacrifice | SorcerySpeed$ True | SpellDescription$ Create a token that's a copy of another target creature you control. It gains haste and "When this creature dies, draw a card." Sacrifice it at the beginning of the next end step. Activate only as a sorcery.
+A:AB$ CopyPermanent | Cost$ R T Discard<1/Card> | ValidTgts$ Creature.Other+YouCtrl | TgtPrompt$ Select another target creature you control | PumpKeywords$ Haste | AtEOT$ Sacrifice | SorcerySpeed$ True | RememberTokens$ True | SubAbility$ DBAnimate | SpellDescription$ Create a token that's a copy of another target creature you control. It gains haste and "When this creature dies, draw a card." Sacrifice it at the beginning of the next end step. Activate only as a sorcery.
+SVar:DBAnimate:DB$ Animate | Defined$ Remembered | Triggers$ Dies | sVars$ TrigDraw | SubAbility$ DBCleanup
```
Comment (Hanmac) [inline forge-gui/res/cardsfolder/j/jaxis_the_troublemaker.txt:6]: is that sVars really needed? i thought we removed the need for svars

**Comment by Hanmac** [comment]
i think i will make a AnimateSubAbility there, but that can be done in later MR

File: forge-gui/res/cardsfolder/j/jaxis_the_troublemaker.txt (hunk 1 above)
Comment (tool4ever) [reply forge-gui/res/cardsfolder/j/jaxis_the_troublemaker.txt]: Right, seems to work fine

**Pull request merged**